src/commands/meal.py

```diff
@@ -0,0 +1,29 @@
+from src.util.const import (
+    BREAKFAST, BREAKFAST_TEMPLATE,
+    DINNER_TEMPLATE,
+    MENU_MSG
+)
+from src.util.util import parse_menu
+from src.database import connect
+from src.util.const import NO_MENU_MSG
+import psycopg2.extras
+from datetime import date
+
+
+def handle_menu(meal):
+    def inner(update, context):
+        """Send the user menu"""
+        # get menu from database
+        conn = connect()
+        cur = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        cur.execute(f"SELECT * FROM {meal} WHERE date = %s;", (date.today(),))
```

Comment:
SQL statements should be extractd into a separate module

Reply:
how to extract template string like this? parameterize the input like date.today()?

Reply:
See https://www.psycopg.org/docs/sql.html